tracker/gioui/patch/button.go

```diff
@@ -0,0 +1,196 @@
+////////////
+// qm210: Had to copy this button because Gioui v0.7.1 makes the Space Bar trigger the last Clicked button again.
+// this interferes too much with the mindset of a music tracker.
+////////////
+
+// SPDX-License-Identifier: Unlicense OR MIT
+
+package patch
+
+import (
+	"image"
+	"time"
+
+	"gioui.org/gesture"
+	"gioui.org/io/event"
+	"gioui.org/io/key"
+	"gioui.org/io/pointer"
+	"gioui.org/io/semantic"
+	"gioui.org/layout"
+	"gioui.org/op"
+	"gioui.org/op/clip"
+)
+
+// Clickable represents a clickable area.
+type Clickable struct {
+	click   gesture.Click
+	history []Press
+
+	requestClicks int
+	pressedKey    key.Name
```

Comment:
We don't need pressedKey as we won't be capturing focus, so the buttons won't receive any keys.